packages/web-react/src/components/UNSTABLE_PartnerLogo/demo/PartnerLogoDefault.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PartnerLogoDemoFactory from './PartnerLogoDemoFactory';
+
+const PartnerLogoDefault = () => (
+  <div style={{ display: 'flex', flexWrap: 'wrap' }}>
```

Comment:
I think we have utility class names for this 🤔

Reply:
Actually, I tried `className="d-flex flex-wrap"`, but the wrap class doesn't work correctly 🤔

Reply:
Ok.